src/common/security/local/context_test.go

```diff
@@ -210,66 +211,73 @@ func TestIsSolutionUser(t *testing.T) {
 func TestHasReadPerm(t *testing.T) {
```

Comment:
Could you refine the test cases after this is merged? 
There should be only be `TestCan` since we have moved away from `HasXXXPerm`

Reply:
ISSUE https://github.com/goharbor/harbor/issues/6881